ground/src/main/java/com/google/android/ground/persistence/local/room/dao/SubmissionMutationDao.kt

```diff
@@ -20,14 +20,13 @@ import androidx.room.Query
 import com.google.android.ground.persistence.local.room.entity.SubmissionMutationEntity
 import com.google.android.ground.persistence.local.room.fields.MutationEntitySyncStatus
 import com.google.android.ground.persistence.local.room.fields.MutationEntityType
-import com.google.android.ground.rx.annotations.Cold
-import io.reactivex.Flowable
+import kotlinx.coroutines.flow.Flow
 
 /** Data access object for database operations related to [SubmissionMutationEntity]. */
 @Dao
 interface SubmissionMutationDao : BaseDao<SubmissionMutationEntity> {
   @Query("SELECT * FROM submission_mutation")
-  fun loadAllOnceAndStream(): Flowable<List<SubmissionMutationEntity>>
+  fun getAllMutations(): Flow<List<SubmissionMutationEntity>>
```

Comment:
Should we append `Flow` to this fun name for consistency with outher flow DAO methods?

Reply:
Works for me! My original thinking was that including the type in the method name is a slight impl detail leak (what if we start using Flobs one day instead of Flows?) but such a change is probably unlikely at this stage and consistency is worth it.

Reply:
I agree! For flows that don't require args think read-only member vals would be more Kotlinic (please cmiiw), in which case we could also remove the `Flow` suffix, for ex:

```
val submissionMutations: Flow<Mutation>
  get() = /* return flow from backing data store */
```

Is there any precedent in examples and docs for naming flows? Don't want to bike shed, if not we can just move forward.

Reply:
props work as well -- we could just do that. the flow lib does use `Flow` suffixes, but that's for specializations of the Flow type (MutableStateFlow) etc. I think let's just use it for now to move forward and we can always rename stuff later

Reply:
Sgtm!